<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TTTR-Convert</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../ui/resource.qrc">
    <normaloff>:/icons/icons/Document-save.png</normaloff>:/icons/icons/Document-save.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="3" column="5">
      <widget class="QPushButton" name="toolButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>convert</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="6">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Output</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Input type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="toolButton_4">
       <property name="text">
        <string>clear inputs</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QComboBox" name="comboBox_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>HDF_TP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Photon_HDF</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1" colspan="4">
      <widget class="QComboBox" name="comboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionOpen_Target">
   <property name="text">
    <string>Open Target</string>
   </property>
  </action>
  <action name="actionEnding_changed">
   <property name="text">
    <string>Ending changed</string>
   </property>
  </action>
  <action name="actionClear_list">
   <property name="text">
    <string>Clear list</string>
   </property>
  </action>
  <action name="actionMake_file">
   <property name="text">
    <string>Make file</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../ui/resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>actionOpen_Target</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>actionEnding_changed</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_4</sender>
   <signal>clicked()</signal>
   <receiver>actionClear_list</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>actionClear_list</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
